gcc/rust/typecheck/rust-tyty.cc

```diff
@@ -53,8 +55,9 @@ BaseType::satisfies_bound (const TypeBoundPredicate &predicate) const
   for (auto &b : probed)
     {
       const Resolver::TraitReference *bound = b.first;
-      bool found = bound->get_mappings ().get_defid ()
-		   == query->get_mappings ().get_defid ();
+      bool found
+	= DefIdComparator::equals (bound->get_mappings ().get_defid (),
+				   query->get_mappings ().get_defid ());
```

Comment:
Is the indentation here bad or is it just GitHub not knowing how to render tabulator characters properly? Assuming the latter given the rendering of `return true;` below.

Reply:
I think the indentation is ok and clang-format did not show any issues. Most likely the tab character rendering is weird. There are 4 space in the beginning of line 58 but 1 tab in the beginning of line 59.